### PR TITLE
Minimum changes required to run bin/gwnr_create_public_events_workflow with py2 and py3

### PR DESCRIPTION
**Pull request by osheamonn**
This is just the minimum changes I had to make to get the workflow working. 

- Celluloid requires python3, so only conditionally import it. 
- Catch ImportError for pyswarm (Maybe make a requirement?)
- `commands` is deprecated since 2.6 in favour of subproces, for now I just remove the unused imports. 
- Allow print to work with 2 and 3
- Use setuptools, since it allows `python setup.py develop`

### CODE REVIEW

File: GWNRTools/Stats/FisherMatrixUtilities.py [hunk 1]
```diff
@@ -26,16 +26,18 @@
...
 import numpy as np
 import scipy as sp
 import string
 import time
 import copy as cp
 
-from pyswarm import pso
-
+try: 
+    from pyswarm import pso
```
Comment (prayush) [inline GWNRTools/Stats/FisherMatrixUtilities.py:38]: just remove `pyswarm`'s import here, its not used in this file anyway.

File: GWNRTools/DataAnalysis/OverlapFunctions.py [hunk 2]
```diff
@@ -23,25 +25,22 @@
...
+from pycbc.waveform import get_td_waveform, get_fd_waveform
+from pycbc.pnutils import nearest_larger_binary_number
+
+from glue import gpstime, git_version
+from glue.ligolw import ligolw
+from glue.ligolw import lsctables
+from glue.ligolw import utils as ligolw_utils
+from glue.ligolw.utils import process as ligolw_process
+import lalsimulation as ls
+import lal
```
Comment (prayush) [inline GWNRTools/DataAnalysis/OverlapFunctions.py:43]:
```
from glue.ligolw import lsctables
import lal
```

Remove all other imports, they are not used below.

File: GWNRTools/Stats/LALInferenceUtilities.py [hunk 3]
```diff
@@ -7,18 +7,23 @@
...
 import scipy as sp
 import string
 import time
 import copy as cp
 
-from pyswarm import pso
+try: 
+    from pyswarm import pso
+except ImportError: 
+    pass
```
Comment (prayush) [inline GWNRTools/Stats/LALInferenceUtilities.py:26]: please remove this import completely, pyswarm isnt really used here

**Comment by osheamonn** [comment]
Updated, thanks!

**Comment by prayush** [comment]
Looks good, thanks @osheamonn! Please squash, if you want fewer commits / want to squash fixups (e.g. there are 2 commits that both deal with excluding imports for `pyswarm`, those should probably be squashed). If you want to keep all commits here as separate, please `reword` the commit messages to not include "fixup"!

**Comment by osheamonn** [comment]
Squashed! I got into the habit of writing "fixup!" since I use to use "git rebase -i --autosquash" which I think automatically marks the commits for squashing.

**Pull request merged**